spec/factories/users.rb

```diff
@@ -3,5 +3,17 @@
     name { Faker::Name.name }
     email { Faker::Internet.email }
     password { Faker::Internet.password }
+
+    trait :admin do
+      role_name { 'admin' }
```

Comment:
You don't need the braces for these traits as that role_name will never change during a factory action.